src/Integration.Vsix/CFamily/ProcessRunner.cs

```diff
@@ -150,7 +150,7 @@ private bool WaitForProcessToFinish(Process process, ProcessRunnerArguments runn
             // true: we might still have timed out, but the process ended when we asked it to
             if (succeeded)
             {
-                LogDebug(CFamilyStrings.MSG_ExecutionExitCode, process.ExitCode);
+                LogMessage(CFamilyStrings.MSG_ExecutionExitCode, process.ExitCode);
```

Comment:
This always prints the exit code(even in case of timeout). And the exit code isn't a helpful message.
Can we do something like:
```
if(process.ExitCode!=0){
LogMessage("Analysis of this file has failed. Please generate a reproducer file *maybe a link to reproducer doc* and share it on *link to our community*);
}
```

Reply:
This message will be a bit problematic, as it's a generic class that runs exe.
Will a message like this be ok? : "Process failed, error code: ..."